### ISSUE TITLE
injection does not seem to work

### ISSUE THREAD

**Issue by JustaGithubGuy** [Body-Alhoha/OpenEctasy#33]
when i put my jar file and click inject it doesnt create a new file

**Comment by Body-Alhoha** [comment]
r u sure?

**Comment by 201122771** [comment]
Yes it doesn't inject.  Jar file provided and webhook and everything, it just says injected successfully and doesn't output the injected jar file.

**Comment by Body-Alhoha** [comment]
could you send a video file of this happening please

**Comment by 201122771** [comment]
https://youtu.be/7KFZ36iqQjQ

**Comment by Body-Alhoha** [comment]
oh well idk i stopped working on this if someone wanna fix this sure

**Comment by orznthedev** [comment]
same what is this??

**Comment by orznthedev** [comment]
when using CLI it says that it could not inject it.

**Comment by Body-Alhoha** [comment]
does it show any error except the fact it did not injecting? try also running the injector with a newer java version, if the injector java version has been released before it wont parse the jar correctly thus not inject

**Comment by orznthedev** [comment]
OMG I FIGURED IT OUT.. you need to use it on spigot plugins and i did paper.. PAPER DOESN'T WORK!!!!!!!!

**Comment by Body-Alhoha** [comment]
oh well nice if u wanna do a PR to make it inject into paper plugins, sure.
